config/kafka/server.properties

```diff
@@ -7,24 +7,29 @@ listener.security.protocol.map=auth-listener-docker-internal:SASL_PLAINTEXT,auth
 sasl.mechanism.inter.broker.protocol=OAUTHBEARER
 
 # Internal docker oauth2 client, which is also used for inter-broker communication.
+# "oauth.client.id" is the valid audience
 listener.name.auth-listener-docker-internal.sasl.enabled.mechanisms=OAUTHBEARER
 listener.name.auth-listener-docker-internal.oauthbearer.sasl.jaas.config=org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required \
-    oauth.client.id="kafka-admin" \
-    oauth.client.secret="kafka-admin-secret" \
```

Comment:
why this change

Reply:
Because oauth.client.id is used for both the valid audience and for getting a token for inter-broker communication, and I would like the audience to be "kafka". Therefore I added a client in keycloak with the client-id kafka, and made this change

Reply:
I have a really bad time reading `config/keycloak/local-development.json` file, but it seems strange that there is a kafka-admin user which is not being used internally by Kafka.

Redpanda, however, uses the `kafka-admin`.
So I think the naming is confusing

Reply:
I changed it so now redpanda uses the `redpanda` client credentials, and kafka setup also uses the new `kafka` client. Which means that the `kafka-admin` is not used internally anymore and could be removed. But I would like to keep it to not break any potential clients using it.

Reply:
I vote to remove it, as clients outside this repo should not be using it?